docs/en/user_guides/3_inference.md

```diff
@@ -0,0 +1,126 @@
+# Inference
+
+We provide demo scripts to inference a given video or a folder that contains continuous images. The source codes are available [here](https://github.com/open-mmlab/mmtracking/tree/dev-1.x/demo/).
+
+Note that if you use a folder as the input, the image names there must be  **sortable** , which means we can re-order the images according to the numbers contained in the filenames. We now only support reading the images whose filenames end with `.jpg`, `.jpeg` and `.png`.
+
+## Inference VID models
+
+This script can inference an input video with a video object detection model.
+
+```
+python demo/demo_vid.py \
+    ${CONFIG_FILE}\
+    --input ${INPUT} \
+    --checkpoint ${CHECKPOINT_FILE} \
+    [--output ${OUTPUT}] \
+    [--device ${DEVICE}] \
+    [--show]
+```
+
+The `INPUT` and `OUTPUT` support both mp4 video format and the folder format.
+
+Optional arguments:
+
+- `OUTPUT`: Output of the visualized demo. If not specified, the `--show` is obligate to show the video on the fly.
+- `DEVICE`: The device for inference. Options are `cpu` or `cuda:0`, etc.
+- `--show`: Whether show the video on the fly.
+
+**Examples:**
+
+Assume that you have already downloaded the checkpoints to the directory `checkpoints/`, your video filename is `demo.mp4`, and your output path is the `./outputs/`
+
+```shell
+python ./demo/demo_vid.py \
+    ./configs/vid/selsa/selsa_faster-rcnn-resnet50-dc5_8x1bs-7e_imagenetvid.py \
+    --input ./demo.mp4 \
+    --checkpoint checkpoints/selsa_faster_rcnn_r101_dc5_1x_imagenetvid_20201218_172724-aa961bcc.pth \
+    --output ./outputs/ \
+    --show
+```
+
+## Inference MOT/VIS models
+
+This script can inference an input video / images with a multiple object tracking or video instance segmentation model.
+
+```shell
+python demo/demo_mot_vis.py \
+    ${CONFIG_FILE} \
+    --input ${INPUT} \
+    [--output ${OUTPUT}] \
+    [--checkpoint ${CHECKPOINT_FILE}] \
+    [--score-thr ${SCORE_THR} \
+    [--device ${DEVICE}] \
+    [--backend ${BACKEND}] \
```

Comment:
backend is not used in visualizatin. Suggest removing this line and remove backend arg in the demo_mot_vis.py